.neutrinorc.js

```diff
@@ -65,6 +65,7 @@ module.exports = {
       'AUTH0_RESPONSE_TYPE',
       'AUTH0_SCOPE',
       'BASE_URL',
+      'DOMAIN',
```

Comment:
We already have `BASE_URL` which is supposed to drive this, but it has been renamed `TASKCLUSTER_ROOT_URL`.